core/micro-services/workflow-core/src/main/scala/edu/uci/ics/amber/storage/result/OpResultStorage.scala

```diff
@@ -0,0 +1,82 @@
+package edu.uci.ics.texera.workflow.common.storage
```

Comment:
Do not use texera. Use edu.uci.ics.amber.core.storage

Reply:
moved